docs/hub/model-card-annotated.md

```diff
@@ -0,0 +1,305 @@
+
+# Bias, Risks, and Limitations
+
+**Section Overview:** This section identifies foreseeable harms and misunderstandings, and both technical and sociotechnical limitations. It also provides information on warnings and potential mitigations.
+
+{{ bias_risks_limitations | default("[More Information Needed]", true)}}
+
+
+_What are the known or foreseeable issues stemming from this model?_
+
+
+## Recommendations
+
+
+{{ bias_recommendations | default("Users (both direct and downstream) should be made aware of the risks, biases and limitations of the model. More information needed for further recommendations.", true)}}
+
+_What are recommendations with respect to the foreseeable issues? This can include everything from “downsample your image” to filtering explicit content._
+
+
+# Training Details
+
+**Section Overview:** This section provides information to describe and replicate training, including the training data, the speed and size of training elements, and the environmental impact of training. This relates heavily to the [Technical Specifications](#technical-specifications) as well, and content here should link to that section when it is relevant to the training procedure.  It is useful for people who want to learn more about the model inputs and training footprint.
+It is relevant for anyone who wants to know the basics of what the model is learning.
+
+## Training Data
+
+
+{{ training_data | default("[More Information Needed]", true)}}
+
+_Write 1-2 sentences on what the training data is. Ideally this links to a Dataset Card for further information. Links to documentation related to data pre-processing or additional filtering may go here as well as in [More Information](#more-information)._
+ 
+
+## Training Procedure [optional]
+
+
+### Preprocessing
+
+
+{{ preprocessing | default("[More Information Needed]", true)}}
+
+
+### Speeds, Sizes, Times
+
+
+{{ speeds_sizes_times | default("[More Information Needed]", true)}}
+
+_Detail throughput, start/end time, checkpoint sizes, etc._
+
+
+# Evaluation
+
+**Section Overview:** This section describes the evaluation protocols, what is being measured in the evaluation, and provides the results.  Evaluation is ideally constructed with factors, such as domain and demographic subgroup, and metrics, such as accuracy, which are prioritized in light of foreseeable error contexts and groups. Target fairness metrics should be decided based on which errors are more likely to be problematic in light of the model use. 
+
+
+## Testing Data, Factors & Metrics
+
+### Testing Data
+
+{{ testing_data | default("[More Information Needed]", true)}}
+
+_Ideally this links to a Dataset Card for the testing data._
+
+### Factors
+
+{{ testing_factors | default("[More Information Needed]", true)}}
+
+_What are the foreseeable characteristics that will influence how the model behaves? This includes domain and context, as well as population subgroups. Evaluation should ideally be **disaggregated** across factors._
```

Comment:
Maybe giving an example on this would be useful

Reply:
Added as a suggestion!

Reply:
Incorporated!